### PR TITLE
Fix Styling

### LINKED ISSUE #1: [FEAT] Issue Templates
We need to set up issue templates. We can copy them over from [LethalAPI.Core](https://github.com/LethalCompany/LethalAPI.Core/tree/development/.github/ISSUE_TEMPLATE)

### PR DESCRIPTION
**Pull request by Redforce04**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
-# LethalAPI.Monsters
+# Warning
```
Comment (MaxWasUnavailable) [inline README.md]: README doesn't need a template mention I think?

**Review by MaxWasUnavailable — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Redforce04) [reply README.md]: Good catch!

**Comment by Redforce04** [comment]
Resolved.

**Review by MaxWasUnavailable — APPROVED** [review]
Approved

**Pull request merged**